lib/pact_broker/client/webhooks/create.rb

```diff
@@ -36,7 +36,11 @@ def call
         attr_reader :http_client
 
         def create_webhook_with_consumer_and_provider
-          webhook_entity = webhook_link.expand(consumer: params.consumer, provider: params.provider).post(request_body)
+          if params.uuid
+            webhook_entity = webhook_link_with_uuid.expand(consumer: params.consumer, provider: params.provider, uuid: params.uuid).put(request_body)
```

Comment:
Remove the `consumer: params.consumer, provider: params.provider` from this expansion. The URL only takes a UUID. The consumer and provider go into the request body.

Reply:
The `put(request_body)` should be `put(request_body_with_optional_consumer_and_provider)`.

Reply:
As above, my implementation looks a little different now that we are working with a new command - essentially I moved all the logic into the `create_webhook_with_optional_consumer_and_provider` command and left everything else as it was